### PR TITLE
Show error message on Framework35

### PR DESCRIPTION
**Pull request by smadala**
## Description
- Show error message when user pass `/Framework:Framework35` as we don't have testhost built for CLR 2.0

## Related issue
https://devdiv.visualstudio.com/DevDiv/_workitems/edit/493289

### CODE REVIEW

File: src/vstest.console/Resources/Resources.resx [hunk 1]
```diff
@@ -276,11 +276,14 @@
   <data name="FileNotFound" xml:space="preserve">
     <value>'{0}' not found.</value>
   </data>
+  <data name="Framework35NotSupported" xml:space="preserve">
+    <value>Framework35 not supported. Use Framework40 or above to run tests in CLR 4.0 "compatibly mode".</value>
```
Comment (singhsarab) [inline src/vstest.console/Resources/Resources.resx]: Suggestion:
Framework35 is not supported. For projects targeting .Net Framework 3.5, please use Framework40 to run tests in CLR 4.0 "compatibility mode".

**Review by singhsarab — APPROVED** [review]
Approving with suggestions.

@cltshivash @pvlakshm please review the change in behaviour

**Comment by pvlakshm** [comment]
Are we erroring out if Framework35 is specified? We should not. Today, we run such tests in CLR 4.0's compat mode. Just call that out as a warning so the user is more aware.

**Comment by smadala** [comment]
@pvlakshm In Tpv1 if user passes Framework35, tests runs in CLR 2.0 process. From VS 15.5 we have changed that behavior to run tests CLR 4.0 process in `compact mode`. If we don't error out user might think his tests running in CLR2.0 process.

As we don't support running tests in CLR 2.0 process, we should remove the `/Framework:Framework35`.

**Comment by smadala** [comment]
>Just call that out as a warning so the user is more aware.

Error makes more awareness. More details refer https://github.com/Microsoft/vstest/issues/1643#issuecomment-399493125

**Pull request merged**